src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -19,8 +19,20 @@ public class RazorViewEngineOptions
             new CSharpCompilationOptions(CodeAnalysis.OutputKind.DynamicallyLinkedLibrary);
         private Action<RoslynCompilationContext> _compilationCallback = c => { };
 
+        public RazorViewEngineOptions()
+        {
+            ViewLocationFormats = new List<string>();
```

Comment:
I didn't want to do that as some code would be inline and some here, but based on @dougbu  comment, since i will be moving some code here to options setup, i can do inline now